libyul/CompilabilityChecker.h

```diff
@@ -43,7 +43,12 @@ namespace solidity::yul
 class CompilabilityChecker
 {
 public:
-	static std::map<YulString, int> run(
+	struct FunctionStackErrorInfo
```

Comment:
If you extract this to the file level, then you don't have to replace everything by `auto`.

Reply:
I think I'd still use ``auto`` then :-). But true, it'll make not using auto less painful.